plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/edit.tsx

```diff
@@ -50,22 +31,15 @@ const Edit = ( {
 		template: TEMPLATE,
 	} );
 
-	const { postType: contextPostType } = context;
-	const isInvalidQueryLoopContext = useIsInvalidQueryLoopContext(
-		clientId,
-		contextPostType
-	);
-	if ( isInvalidQueryLoopContext ) {
-		return (
-			<div { ...blockProps }>
-				<Warning>
-					{ __(
-						'The Product Reviews block requires a product context. When used in a Query Loop, the Query Loop must be configured to display products.',
-						'woocommerce'
-					) }
-				</Warning>
-			</div>
-		);
+	const { hasInvalidContext, warningElement } =
+		useQueryLoopProductContextValidation( {
+			clientId,
+			postType: context.postType,
+			blockName: 'Product Reviews',
```

Comment:
Same as above, should this be translated?

Reply:
Agreed! Updated in [df30e31](https://github.com/woocommerce/woocommerce/pull/56431/commits/df30e313426936eb9561997494598a3321004bfc).